commit 869790ec4a6fac85d6a8653e747170ed66528cff
Author: Kamal Mostafa <kamal@canonical.com>
Date:   Tue Nov 13 13:54:34 2012 -0800

    UBUNTU: SAUCE: input: Cypress PS/2 Trackpad fix input_mt_init_slots() call
    
    Add new third arg to fix input_mt_init_slots() call.
    
    Signed-off-by: Kamal Mostafa <kamal@canonical.com>

diff --git a/drivers/input/mouse/cypress_ps2.c b/drivers/input/mouse/cypress_ps2.c
index 317aac9..1e23252 100644
--- a/drivers/input/mouse/cypress_ps2.c
+++ b/drivers/input/mouse/cypress_ps2.c
@@ -411,7 +411,7 @@ static int cypress_set_input_params(struct input_dev *input,
 		input_set_abs_params(input, ABS_MT_POSITION_Y, 0, cytp->tp_max_abs_y, 0, 0);
 		input_set_abs_params(input, ABS_MT_PRESSURE, 0, 255, 0, 0);
 
-		ret = input_mt_init_slots(input, CYTP_MAX_MT_SLOTS);
+		ret = input_mt_init_slots(input, CYTP_MAX_MT_SLOTS, 0);
 		if (ret < 0) {
 			return ret;
 		}
